src/app/pages/send/send-crypto-asset-form/family/stacks/components/stacks-recipient-field.tsx

```diff
@@ -1,7 +1,7 @@
-import { fetchNameOwner } from '@leather.io/query';
+import { fetchStacksNameOwner } from '@leather.io/query';
 
 import { RecipientField } from '../../../components/recipient-fields/recipient-field';
 
 export function StacksRecipientField() {
-  return <RecipientField bnsLookupFn={fetchNameOwner} />;
+  return <RecipientField bnsLookupFn={fetchStacksNameOwner} />;
```

Comment:
I took a look into `fetchStacksNameOwner` from curiosity and see that it's almost exactly the same as `fetchBtcNameOwner` but we check for a different field.

Can we:
- make these more like the original `fetchNameOwner` to share code?
- get rid of the old `fetchNameOwner` we don't use now?